app/src/main/java/com/yacgroup/yacguide/PreferencesActivity.kt

```diff
@@ -25,10 +25,14 @@ import androidx.core.content.ContextCompat
 import android.view.View
 import android.widget.Button
 import android.widget.CheckBox
+import android.widget.Toast
+import com.yacgroup.yacguide.database.DatabaseWrapper
+import com.yacgroup.yacguide.utils.DialogWidgetBuilder
 
 class PreferencesActivity : BaseNavigationActivity() {
 
     private lateinit var _customSettings: SharedPreferences
+    private lateinit var _db: DatabaseWrapper
```

Comment:
Minor: It is more convenient for developers if the order of members is the same as they are initialized. So please put declaration `_db` one line up.

Reply:
Good to know, thanks.